app/src/main/java/org/jellyfin/mobile/setup/ComposeFragment.kt

```diff
@@ -0,0 +1,46 @@
+package org.jellyfin.mobile.setup
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.ui.platform.ComposeView
+import androidx.core.view.ViewCompat
+import androidx.fragment.app.Fragment
+import org.jellyfin.mobile.MainViewModel
+import org.jellyfin.mobile.databinding.FragmentComposeBinding
+import org.jellyfin.mobile.ui.screens.connect.ConnectScreen
+import org.jellyfin.mobile.ui.utils.AppTheme
+import org.jellyfin.mobile.utils.Constants
+import org.jellyfin.mobile.utils.applyWindowInsetsAsMargins
+import org.koin.androidx.viewmodel.ext.android.sharedViewModel
+
+class ComposeFragment : Fragment() {
```

Comment:
Why not call it ConnectFragment like before? It still only does connecting

Reply:
For now, yes. It'll probably handle more in the future or be outright removed but I can of course rename it.

Reply:
If we add more to it in the future we can rename it at that point. I'd imagine the webview will be embedded directly in a compose screen using https://google.github.io/accompanist/webview/

Reply:
Oh, I didn't know there was an Accompanist library for this. I'd just have used `AndroidView`. Thanks for the link!

Reply:
I used that in one of my own apps after I used an AndroidView myself and ran into an issue :-)